src/user-guide/interventions/index.rst

```diff
@@ -0,0 +1,433 @@
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: set-task-outputs.gui.gif
+         :width: 75%
+
+      By default this sets the "succeeded" output, press the pencil icon next
+      to the trigger command to specify a different output.
+
+   .. tab-item:: Tui
+      :sync: tui
+
+      .. image:: set-task-outputs.tui.gif
+         :width: 75%
+
+      By default, this sets the "succeeded" output. Use ``cylc set --output``
+      to specify a different output.
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc set <workflow>//<task>
+
+      By default, this sets the "succeeded" output. Use the ``--output`` option
+      to specify a different output.
+
+You can also :ref:`set task prerequisites <interventions.set-task-prerequisites>`.
+Should I set task outputs or prerequisites?
+
+* If you set a task's outputs, then tasks downstream of it may start to run.
+* If you set a task's prerequisites, this will not happen.
+
+
+.. _interventions.set-task-prerequisites:
+
+Set Task Prerequisites
+----------------------
+
+.. workflow config:
+
+   [scheduler]
+       allow implicit tasks = True
+   
+   [scheduling]
+       [[graph]]
+           R1 = """
+               a => z1 & z2
+               b1 => b2 => z1 & z2
+               c => z1 & z2
+
+               # set prereqs "1/b2:succeeded" and "1/c:succeeded" on 1/z1
+               # set prereqs "all" on 1/z2
+           """
+   
+   [runtime]
+       [[b1]]
+           script = sleep 600
+       [[c]]
+           script = false
+
+:Example:
+   A task is not running yet, because one or more of its
+   :term:`prerequisites <trigger>` are not satisfied (e.g. upstream tasks
+   have not run yet or have failed). I want it to ignore one or more of these
+   prerequisites.
+
+:Solution:
+   Set the task's prerequisites as satisfied.
+
+.. note::
+
+   If you want the task to run right away, then trigger it using the same
+   intervention as :ref:`interventions.re-run-a-task`.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: set-task-prerequisites.gui.gif
+         :width: 75%
+
+   .. tab-item:: CLI
+      :sync: cli
+
+      .. code-block:: console
+
+         $ cylc set \
+         >   --pre <prereq-cycle>/<prereq-task>:<prereq-output> \
+         >   <workflow>//<cycle><task>
+
+You can also :ref:`set task outputs <interventions.set-task-outputs>`.
+Should I set task outputs or prerequisites?
+
+* If you set a task's outputs, then tasks downstream of it may start to run.
+* If you set a task's prerequisites, this will not happen.
+
+
+Set a Switch Task
+-----------------
+
+.. workflow config:
+
+   [scheduling]
+       cycling mode = integer
+       initial cycle point = 1
+       runahead limit = P1
+       [[graph]]
+           P1 = """
+               start => switch
+               switch:normal? => normal
+               switch:alternate? => alternate
+               normal | alternate => end
```

Comment:
This has the problem that @MetRonnie and I flagged on the optional-outputs-extension PR. It looks nice, but novice users might not understand that the `normal` thing in `switch:normal` is not the same thing as in `=> normal`.  

Not critical though, can punt it to https://github.com/cylc/cylc-flow/issues/6085

Reply:
IMO, it's clearer this way than the alternative but open to reasonable suggestions.

Reply:
No need to change it on this PR anyway.